src/pudl/settings.py

```diff
@@ -213,10 +213,12 @@ class Eia860Settings(GenericDatasetSettings):
 
     data_source: ClassVar[DataSource] = DataSource.from_id("eia860")
     eia860m_data_source: ClassVar[DataSource] = DataSource.from_id("eia860m")
-    eia860m_date: ClassVar[str] = eia860m_data_source.working_partitions["year_month"]
-
     years: list[int] = data_source.working_partitions["years"]
-    eia860m: bool = True
+    eia860m_year_months: list[str] = eia860m_data_source.working_partitions[
+        "year_months"
+    ]
```

Comment:
I was wondering that... i think it might make sense but they are intertwined so I hesitated. This old boolean was used to determine whether one month of eia860m was incorporated into the core 860 assets.

If we want to make a new settings class I think we'd need to keep the old eia860m boolean in the 860 settings class. I was thinking the 860 and 860m classes should talk to each other which imo would be messy but they can both go look at the working partitions from sources separately.